insights/client/phase/v1.py

```diff
@@ -112,7 +112,12 @@ def pre_update(client, config):
         sys.exit(constants.sig_kill_ok)
 
     if config.checkin:
-        checkin_success = client.checkin()
+        try:
+            checkin_success = client.checkin()
+        except Exception as e:
```

Comment:
Exceptions are not Pokémon, you don’t have to catch ’em all. What happened there in the _checkin_ method that caused the need for try-except? Wouldn’t catching a specific type be better? Why isn’t it enough that the exception is caught by the phase, printing “Fatal error”?

Reply:
This PR is a quick fix intended to be consistent with the other case in which this error occurs. This whole logic set needs to be reworked anyway -- I'll attend to it.